test/util/util.go

```diff
@@ -1110,7 +1110,7 @@ func SetNodeCondition(ctx context.Context, k8sClient client.Client, node *corev1
 	}, Timeout, Interval).Should(gomega.Succeed(), "Failed to set node condition %s to %s for node %s", newCondition.Type, newCondition.Status, node.Name)
 }
 
-func ExpectLocalQueueUsageToBe(ctx context.Context, k8sClient client.Client, lqKey client.ObjectKey, comparator string, compareTo any) {
+func ExpectLocalQueueFairSharingUsageToBe(ctx context.Context, k8sClient client.Client, lqKey client.ObjectKey, comparator string, compareTo any) {
```

Comment:
Let me cherry-pick https://github.com/kubernetes-sigs/kueue/pull/6406 before this PR.

Reply:
Finally, we can came back here.
@PBundyra could you rebase this PR?